installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -526,15 +531,20 @@ protected void deleteSaasBoostInstallation() {
         List<LinkedHashMap<String, Object>> tenants = getProvisionedTenants();
         for (LinkedHashMap<String, Object> tenant : tenants) {
             outputMessage("Deleting AWS SaaS Boost tenant " + tenant.get("id"));
-            deleteProvisionedTenant(tenant);
+            if ((Boolean) tenant.get("active")) {
+                LOGGER.debug("Deleting active tenant", tenant);
+                deleteProvisionedTenant(tenant);
+            } else {
+                LOGGER.debug("Not deleting inactive tenant: ", tenant);
```

Comment:
We should call delete for _all_ tenants regardless of their active status. If a tenant as been "disabled" by an admin, their infrastructure is still provisioned and we need to clean it up before we can delete the rest of the environment.